src/lib/metricsReporter.js

```diff
@@ -0,0 +1,44 @@
+const os = require('os')
+const metricsPrometheus = require('./metricsPrometheus')
+
+class MetricsReporter {
+  constructor(config) {
+    this.hostname = os.hostname()
```

Comment:
We're no longer using `hostname` tag on our metrics. Remove this, please.

Reply:
alright! will remove from go client as well

Reply:
Great!

This solves the comment about clientHost as well, right?

Reply:
yes

Reply:
done